test/boost/group0_cmd_merge_test.cc

```diff
@@ -12,11 +12,53 @@
 #include "service/migration_manager.hh"
 #include "service/raft/raft_group_registry.hh"
 #include "service/raft/group0_state_machine.hh"
+#include "service/raft/group0_state_machine_merger.hh"
 #include "idl/experimental/broadcast_tables_lang.dist.hh"
 #include "idl/experimental/broadcast_tables_lang.dist.impl.hh"
 #include "idl/group0_state_machine.dist.hh"
 #include "idl/group0_state_machine.dist.impl.hh"
 #include "utils/error_injection.hh"
+#include "test/lib/expr_test_utils.hh"
+
+const auto OLD_TIMEUUID = utils::UUID_gen::get_time_UUID(std::chrono::system_clock::time_point::min());
+
+static service::group0_command create_command(utils::UUID id) {
+    auto mut = canonical_mutation{mutation{db::system_keyspace::group0_history(), partition_key::make_empty()}};
+
+    return service::group0_command {
+        .change{service::write_mutations{std::vector<canonical_mutation>{mut}}},
+        .history_append{db::system_keyspace::make_group0_history_state_id_mutation(
+                        id, std::nullopt, "test")},
+        .new_state_id = id,
+        .creator_addr{},
+        .creator_id{raft::server_id::create_random_id()},
+    };
+}
+
+SEASTAR_TEST_CASE(test_group0_state_machine_merger_timeuuid_order) {
+    // https://github.com/scylladb/scylladb/issues/14568
```

Comment:
We usually just reference the issue number with `#`, like this:
```
Ref #14568
```

and below:
```
From #7729
```

Reply:
Also, a better issue (or issues -- there are two in this case) to reference is those that described the bugs which were then solved, and this test is a regression test for the two issues. (The comment should explicitly say that this is a regression test for #X and #Y)